src/app/dao/tests/PartDAO.spec.ts

```diff
@@ -212,15 +228,13 @@ describe('PartDAO', () => {
             // Then it should succeed
             await expectAsync(result).toBeResolvedTo();
         });
-        it('should forbid deleting a non-started part if owner is still observing it and has not timed out', async() => {
-            // Given a part with its owner
+        it('should forbid deleting a started part if the owner has timed out', async() => {
```

Comment:
should forbid deleting a started part even if owner has timed out
should forbid deleting a started part when the owner has not timed out

Ou encore 
should forbid deleting a started part (creator timed out|creator didnt timed outà ?

Reply:
reformulé et corrigé, y'avait de l'inversion de commentaire vs. ce que les tests faisaient :o